### DISCUSSION TITLE
Why is a new instance of the player being launched?

### DISCUSSION THREAD

**Discussion by RiKkzu** [smplayer-dev/smplayer#805] [General]
I am writing through a translator, if my words are not clear, I'm sorry.
Situation: A video from YouTube is being played in SMPlayer. I am writing a command in the console "smplayer -add-to-playlist https://www.youtube.com/watch?v =...". A new instance of SMPlayer opens and plays the video in parallel with the first video. I want to add a video to the current playlist, not watch two videos at the same time. How can this be done?

**Comment by smplayer-dev** [comment]
Be sure the option "use only one instance" in preferences -> interface -> instances is enabled.
